.github/workflows/dependency-check.yml

```diff
@@ -0,0 +1,32 @@
+name: Suppressions
+
+on:
+  schedule:
+    # At 12 midnight and every 3rd hour from 10am through 11pm
```

Comment:
Could the first update start from 7am, just before the nightly pipelines are executed?
And, doing regular updates during the day may trigger master builds out of intent, while some other builds are in progress. Perhaps, it might be better like just 5pm, midnight and 7am?